data/lib/http_emitter.py

```diff
@@ -0,0 +1,17 @@
+import itertools
+import requests
+import uuid
+
+import lib.log_generator as log_generator
+
+ENDPOINT = "http://127.0.0.1:3000"
+
+for _ in itertools.repeat(None, 1000):
```

Comment:
💎 Ruby!

I think the more standard way would be
```
for i in range(100):
``` 
Pretty sure a python linter would not complain about having a un-used variable in this context